pkg/subnet/config.go

```diff
@@ -0,0 +1,51 @@
+// This file is part of MinIO Kubernetes Cloud
+// Copyright (c) 2019 MinIO, Inc.
```

Comment:
```suggestion
// Copyright (c) 2020 MinIO, Inc.
```

Please check in other files